packages/supervisor/src/bin/supervisor.ts

```diff
@@ -16,10 +16,10 @@ import { LifeCycleController } from "../lib/lifecycle-controller";
 const config: LifeCycleConfig = {
     makeSnapshotOnError: false
 };
-const lcdai: LifecycleDockerAdapterInstance = new LifecycleDockerAdapterInstance();
+const instanceAdapter = getInstanceAdapter(process.env.RUN_WITHOUT_DOCKER === "true");
```

Comment:
This should be only set by config values and command line arguments, not env vars. A user may run `sth` without knowing that such an env var was set somewhere.

Reply:
fixed